doc/how-to-build.md

```diff
@@ -119,17 +119,29 @@ Makefile rules:
 ### Troubleshoot Testing
 
 **Set Nginx Path**
+
 - If you run in to an issue `Error unknown directive "lua_package_path" in /API_ASPIX/apisix/t/servroot/conf/nginx.conf`
 make sure to set openresty as default nginx. And export the path as below.
+
 * export PATH=/usr/local/openresty/nginx/sbin:$PATH
     - Linux default installation path:
         * export PATH=/usr/local/openresty/nginx/sbin:$PATH
     - OSx default installation path via homebrew:
         * export PATH=/usr/local/opt/openresty/nginx/sbin:$PATH
 
 **Run Individual Test Cases**
+
+Because the test case of `apisix` depends on the https://github.com/api7/test-toolkit test kit. Therefore, before running a single test case, you need to execute `make test` to automatically download the test kit or manually clone the `test-toolkit` to the `apisix/t/` directory:
+
+```shell
+git clone https://github.com/api7/test-toolkit/ t/toolkit
```

Comment:
Do we need this after https://github.com/apache/apisix/pull/2934?
And this command is incorrect. Running it will get `fatal: destination path 't/toolkit' already exists and is not an empty directory.`

Reply:
Oh, I think we can remove it.

Reply:
What I think is necessary, before running a single test case, if we have not executed the `make test` command, then `t/toolkit` is an empty directory. Therefore, we need to explain how to download this test kit. At the same time, this is also to show that our test cases depend on the `test-toolkit` package. I will add the reason why this command `git clone https://github.com/api7/test-toolkit/t/toolkit` failed later.

Reply:
> Oh, I think we can remove it.Let user clone it is very danger

Oh, this is also a method. We can remove it if needed.

Reply:
@Firstsawyou 
But the doc already requires user to run `make test`.

Reply:
Understand, this pr seems to be redundant, we can turn it off.